packages/viewer/src/rendering/remote.ts

```diff
@@ -20,6 +22,15 @@ export interface FrameResponse {
 
 export type RemoteRenderer = FrameRenderer<FrameRequest, FrameResponse>;
 
+export const easingMap = {
+  linear: vertexvis.protobuf.stream.EasingType.EASING_TYPE_LINEAR,
+  'ease-out-cubic': vertexvis.protobuf.stream.EasingType.EASING_TYPE_LINEAR,
+  'ease-out-quad': vertexvis.protobuf.stream.EasingType.EASING_TYPE_LINEAR,
+  'ease-out-quart': vertexvis.protobuf.stream.EasingType.EASING_TYPE_LINEAR,
+  'ease-out-sine': vertexvis.protobuf.stream.EasingType.EASING_TYPE_LINEAR,
+  'ease-out-expo': vertexvis.protobuf.stream.EasingType.EASING_TYPE_LINEAR,
```

Comment:
is it intentional for all of these to be set to `vertexvis.protobuf.stream.EasingType.EASING_TYPE_LINEAR`?

Reply:
Nope. Haha. Will fix